content/spin/v2/rdbms-storage.md

```diff
@@ -74,8 +80,19 @@ The Python SDK doesn't currently surface the MySQL or PostgreSQL APIs.
 
 {{ startTab "TinyGo"}}
 
-The Go SDK doesn't currently surface the MySQL or PostgreSQL APIs.
+MySQL functions are available in the `github.com/fermyon/spin/sdk/go/v2/mysql` package, and PostgreSQL in `github.com/fermyon/spin/sdk/go/v2/pg`. [See Go Packages for reference documentation.](https://pkg.go.dev/github.com/fermyon/spin/sdk/go/v2).
+
+The package follows the usual Go database API. Use `Open` to return a connection to the database of type `*sql.DB` - see the [Go standard library documentation](https://pkg.go.dev/database/sql#DB) for usage information.
```

Comment:
Do we want to paste an example/snippet of either/both MySQL and PG in Go (a la the Rust tab?)